R/sjTabItemAnalysis.R

```diff
@@ -139,6 +139,12 @@ sjt.itemanalysis <- function(df,
   # check encoding
   encoding <- get.encoding(encoding, df)
 
+  # convert ordered factors to labbeled
+  if (all(sapply(df, is.ordered))) df <- sjlabelled::as_labelled(df)
```

Comment:
It would be nice to change ordered factors even if not all are variables are ordered factors, e.g.

```r
o <- sapply(df, is.ordered)
if (any(o)) df[o] <- sjlabelled::as_labelled(df[o])
````

or so (untested code-chunk above, but I guess the idea is clear?)